test/testlib/minikube_utilities.go

```diff
@@ -219,10 +219,22 @@ func InjectTestVersion(t *testing.T, options *helm.Options) {
 	options.SetValues["nuodb.image.tag"] = image.Nuodb.Image.Tag
 }
 
+func InjectSSLMode(t *testing.T, options *helm.Options) {
```

Comment:
Does this logic also rely on `nuodocker` internally modifying its connection URL from https to http if ssl is disabled?

I don't believe it does that yet - but should shortly.